src/renderer/components/FtSearchFilters/FtSearchFilters.vue

```diff
@@ -248,7 +256,28 @@ watch(searchFilterValueChanged, (value) => {
 function hideSearchFilters() {
   store.dispatch('hideSearchFilters')
 }
+function searchWithFilters() {
+  const queryTextElement = document.querySelector('.searchInput input')
```

Comment:
Ideally, you shouldn't use `document.querySelector`. The appropriate way to do this would be to add a value to the utils store for the search query and pull it from there. Perhaps under `searchSettings`: https://github.com/FreeTubeApp/FreeTube/blob/1d1920f2d9a4f7391f61107f0f3be4489667f7a5/src/renderer/store/modules/utils.js#L46-L52